### PR TITLE
Open Results in External Program

### PR DESCRIPTION
**Pull request by mubaidr**
This pull requests implements "Open in external command" for results. 

Corresponding issue/request: : https://github.com/mtxr/vscode-sqltools/issues/259

----

Thank you for your contribution! 
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [ ] You have made the needed changes to the docs
- [x] You have written a description of what is the purpose of this pull request above

### CODE REVIEW

File: packages/plugins/connection-manager/extension.ts [hunk 1]
```diff
@@ -344,6 +345,38 @@ export default class ConnectionManagerPlugin implements IExtensionPlugin {
...
+      };
+    } else if (arg.requestId) {
+      // used context menu inside of a view
+      const { ...rest } = arg;
+      opt = rest;
+    }
+
+    if (!opt || !opt.requestId) throw "Can't find active results view";
+
+    var filename = file({
```
Comment (mtxr) [inline packages/plugins/connection-manager/extension.ts:372]: Can we make it open both JSON and CSV as we do here: https://github.com/mtxr/vscode-sqltools-pro/blob/f7b12ff472fb27eddcc5f05ba150242c15fdf394/packages/plugins/connection-manager/extension.ts#L325

**Comment by mtxr** [comment]
@mubaidr great work BTW! It will be awesome!

File: packages/plugins/connection-manager/extension.ts (hunk 1 above)
Comment (mubaidr) [reply packages/plugins/connection-manager/extension.ts:372]: Well, I had "Open with Excel" in my mind, that is why only `csv` option. I will update this part to add `json` option. 

Also, I think `json` should open in vscode and `csv` should open in external program (i.e. Excel etc)?

**Comment by mtxr** [comment]
You can use the same API. It will open on system default program, it’s ok.

Thanks again!

**Comment by mubaidr** [comment]
Updates:

- Open command has option to open as `csv` or `json`
- Open in external program works fine now (used `vscode.env.openExternal`)
- Added open button to the results window footer.

**Comment by mtxr** [comment]
@mubaidr AWESOME! thanks!

**Pull request merged**